build-system/app.js

```diff
@@ -933,6 +933,7 @@ app.get('/adzerk/*', (req, res) => {
   fs.readFileAsync(filePath).then(file => {
     res.setHeader('Content-Type', 'application/json');
     res.setHeader('AMP-template-amp-creative', 'amp-mustache');
```

Comment:
I thought we already changed to
`AMP-Ad-Template-Extension: amp-mustache `

Reply:
We did, but I must've forgotten to change it here.

Reply:
and you can remove this line now

Reply:
Done.